features/search/components/OldProfileSearchResultsList.tsx

```diff
@@ -102,9 +115,17 @@ const $sectionTitleContainer: ThemedStyle<ViewStyle> = ({
   }),
 });
 
-const $sectionTitleSpacing: ThemedStyle<TextStyle> = ({ spacing }) => ({
-  marginBottom: spacing.xs,
-  marginTop: spacing.lg,
+const $sectionTitleSpacing: ThemedStyle<ViewStyle> = ({ spacing }) => ({
+  ...Platform.select({
+    default: {
+      marginBottom: spacing.sm,
+      marginTop: spacing.xl,
+    },
+    android: {
+      marginBottom: spacing.md,
+      marginTop: spacing.lg,
+    },
```

Comment:
Maybe totally wrong but I don't think we should have specific for Android

Reply:
was just copying what wsa there! I'm sure you're right

Reply:
going to leave this one styled the same as we'll be redesigning this screen, but removed conditionals from new work